examples/webpack/webpack.config.js

```diff
@@ -36,7 +37,8 @@ module.exports = {
               extract: extractCSS,
               importReact: false,
               extensions: ['.js', '.jsx', '.ts', '.tsx', '.customjsx'],
-              babelPlugins: ['typescript'],
+              parserBabelPlugins: ['typescript'],
```

Comment:
Can we update the [docs](https://compiledcssinjs.com/docs/pkg-webpack-loader) with these extra configurable options?